README.md

```diff
@@ -26,14 +26,30 @@ pipenv sync --dev
 
 ## Running tests
 
-To run tests, arangodb must be running locally on the default port with default root credentials. You can run arangodb using docker-compose with `docker-compse up`.
+To run tests, `arangodb` must be running locally on the default port with default root credentials.
+You can run `arangodb` using docker-compose with `docker-compse up`. However, we have found that
+running `arangodb` outside the docker container is ~10x faster for reasons that are currently
```

Comment:
On Mac at least, this is likely to do with reads/writes being slowish with the docker desktop VM

Reply:
I'm on ubuntu *shrug*

Reply:
Although to be specific I'm running an ubuntu VM in an ubuntu host. Maybe the VM + docker combination slows things down for some reason? Although I don't recall anything like this when I've used mongo in docker containers for tests in other repos